### DISCUSSION TITLE
Health check for GraphQL

### DISCUSSION THREAD

**Discussion by corysimmons** [hasura/graphql-engine#8623] [General]
Super low priority feat request, but if it's easy to add, it'd be nice during the early stages of a project to have a way to see if your GraphQL client is getting a response from your Hasura endpoint without having to worry about auth.

`/healthz` REST endpoint exists for general health check but I'd love to be able to test a GraphQL connection with something that completely bypasses all auth like

```gql
query HealthCheck {
  healthz {
    status
    response_time
  }
}
```

**Comment by meetzaveri** [comment]
Hello Cory,

Thanks for reaching out to us in discussions. We love your idea as feature request.

We would recommend you to create a github issue on this with subject of feature request and follow it. That would be a good starting point and our team would be aware of it for tracking it in the future.

**Reply by corysimmons** [reply]
Ah sorry I made a new Issue for it https://github.com/hasura/graphql-engine/issues/8631 and then realized I could have clicked this link
![image](https://user-images.githubusercontent.com/224636/175938456-182e23f7-4b84-4d02-a4c1-462c8d6748b6.png)

And now I can't delete this Discussion. 🤪

**Reply by meetzaveri** [reply]
No worries. It's totally, fine. We can delete this discussion after some time.
